plugins

```diff
@@ -120,6 +120,7 @@ class Plugins < Thor
   option :source,      aliases: '-s',    type: :string,                 default: './schemas', desc: 'Source folder containing the schemas'
   option :destination, aliases: '--dest', type: :string,                   default: './json_schemas', desc: 'Destination folder where the schemas will be written'
   option :skip_custom_annotations, aliases: '--skip-custom-annotations', type: :boolean,                   default: false, desc: 'Skip custom annotations'
+  option :allow_auto_fields, aliases: '--allow-auto-fields', type: :boolean,                   default: false, desc: 'Do not make a field require if it has auto: true'
```

Comment:
nitpick - should be `required if it has auto: true` instead of `require if it has auto: true`?